jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/params/GeneralParams.scala

```diff
@@ -71,6 +71,12 @@ trait GeneralParams extends Params {
   val missing = new FloatParam(this, "missing", "the value treated as missing")
 
   /**
+    * the interval to check whether numCores are sufficient. default: 30 minutes
```

Comment:
Consider giving a little bit of context on when the cores are sufficient.

Reply:
Done.